examples/cpp/13_native_cuda_kernels/main.cpp

```diff
@@ -31,7 +31,6 @@ int main(int argc, const char **argv) {
   occa::memory o_ab = device.malloc<float>(entries);
 
   // Compile a regular CUDA kernel at run-time
-  occa::properties kernelProps;
```

Comment:
Oops, the `occa::properties` above should be `deviceProps` and not `kernelProps`

Reply:
Ah, yes. I was blindly following compiler warnings, but I have the proper fix now.